AutomatedTesting/Gem/PythonTests/editor/EditorScripts/Menus_FileMenuOptions.py

```diff
@@ -38,7 +38,7 @@ def Menus_FileMenuOptions_Work():
         ("Save As",),
         ("Save Level Statistics",),
         ("Edit Project Settings",),
-        #("Edit Platform Settings",),   Temporarily disabled due to https://github.com/o3de/o3de/issues/6604
+        ("Edit Platform Settings",),   #Temporarily disabled due to https://github.com/o3de/o3de/issues/6604
```

Comment:
If this is fixed now, then you can actually remove that comment

Reply:
I forgot to save the file before committing! Done now.